### PR TITLE
[fabric-website] Theme picker renders MDL2 styles.

### PR DESCRIPTION
**Pull request by jdhuntington**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8902)

### CODE REVIEW

**Comment by Jahnp** [comment]
@jdhuntington I'm not sure I understand the impact of this without visualizing it--a bit hard to parse the result from the code alone. Does this render MDL2 visuals by default, or provide it as an option?

**Comment by jdhuntington** [comment]
@Jahnp this fixes the experience seen with the theme picker here

Screen Shot 2019-05-01 at 09 48 06

Couple notes:
 - Fluent is the default
 - This PR does not add the theme picker; @ecraig12345 added that already, but just fixes the rendering to show MDL2 correctly. As committed, it's half-MDL2/half-fluent.

**Comment by Jahnp** [comment]
@jdhuntington ahhh gotcha, makes sense--thanks for clarifying! And, sorry for the delay on reviewing. Looks good to me!
